bentoml/_internal/io_descriptors/image.py

```diff
@@ -116,16 +117,12 @@ async def from_http_request(self, request: Request) -> "PIL.Image":
     async def to_http_response(
         self, obj: t.Union["np.ndarray", "PIL.Image.Image"]
     ) -> Response:
-        if isinstance(obj, np.ndarray):
-            pass
-        elif isinstance(obj, PIL.Image.Image):
-            obj = np.array(obj)
-        else:
+        if not isinstance(obj, [np.ndarray, PIL.Image.Image]):
```

Comment:
this is causing an error in tests:
```
TypeError: isinstance() arg 2 must be a type or tuple of types
```
let's make sure lint and existing tests all pass before merging going forward